WcaOnRails/app/views/admin/edit_person.html.erb

```diff
@@ -22,7 +22,7 @@
       <%= f.input :wca_id, as: :user_ids, persons_table: true, only_one: true %>
       <div id="person-fields">
         <%= f.input :name %>
-        <%= f.input :countryId, collection: Country.all_real.map(&:id), label_method: lambda { |id| Country.find(id).name } %>
+        <%= f.input :countryId, collection: Country.real.pluck, label_method: lambda { |id| Country.find(id).name } %>
```

Comment:
You have so in more places, but as this was done in one commit it should be easy to find.